Add a forward declaration for struct sockaddr_in, which is only passed in
via pointers, but makes the compiler happy.

--- src/util/pty/libpty.h	Tue Dec 21 12:00:50 1999
+++ src/util/pty/libpty.h	Tue Dec 21 12:01:04 1999
@@ -32,6 +32,8 @@
 #define PTY_UTMP_USERNAME_VALID (0x2)
 #ifdef __STDC__ /* use prototypes */
 
+struct sockaddr_in;
+
 long pty_init(void);
 long pty_getpty ( int *fd, char *slave, int slavelength);
 
